test/functional/rpc_rawtransaction.py

```diff
@@ -478,6 +478,20 @@ def transaction_version_number_tests(self):
         decrawtx = self.nodes[0].decoderawtransaction(rawtx)
         assert_equal(decrawtx['version'], 0x7fffffff)
 
```

Comment:
Question: If we are removing the "signedness" of the tx.nVersion, should we think about removing the test cases that made sense for the signed nVersion?

Reply:
I think it makes sense to leave them in to make sure that everything still matches previous versions.